protocol.html

```diff
@@ -369,6 +369,9 @@ <h3 property="schema:name skos:prefLabel">Terminology</h3>
                     <dt about="#root-container" property="skos:prefLabel" typeof="skos:Concept"><dfn id="root-container">root container</dfn></dt>
                     <dd about="#root-container" property="skos:definition">A root container is a container resource that is at the highest level of the collection hierarchy.</dd>
 
+                    <dt about="#authoritative-information" property="skos:prefLabel" typeof="skos:Concept"><dfn id="authoritative-information">authoritative information</dfn></dt>
+                    <dd about="#authoritative-information" property="skos:definition">Authoritative information encompasses data about resources described by means of RDF statements [<cite><a class="bibref" href="#bib-rdf11-concepts">RDF11-CONCEPTS</a></cite>] that can be supplied by clients, servers, and intermediaries.</dd>
```

Comment:
The definition feels like it could use a bit more detail on what it means for this information to be authoritative. Does it mean that consumers should be able to rely on its accuracy? Can any data about a resource provided by anyone be considered authoritative since clients, servers, and intermediaries are mentioned?

Reply:
https://github.com/solid/specification/pull/352#discussion_r762578541